persistence-modules/pom.xml

```diff
@@ -113,7 +113,7 @@
         <module>spring-jpa-2</module>
         <module>spring-jpa-3</module>
         <module>spring-jdbc</module>
```

Comment:
Have you added some space before <module>spring-jdbc-2</module> ?
Do we need it?

Reply:
yes it was wrongly indented earlier